build/teamcity/internal/release/build-and-publish-patched-go/impl-fips.sh

```diff
@@ -26,7 +26,6 @@ sed -i "s/go mod tidy/go mod tidy -go=1.16/g" scripts/create-secondary-patch.sh
 cd go/src
 # Apply the CRL patch
 patch -p2 < /bootstrap/diff.patch
-patch -p2 < /bootstrap/6446af9.diff
```

Comment:
What's the context on this dropping?

Reply:
Internal slack [thread](https://cockroachlabs.slack.com/archives/CJ0H8Q97C/p1713821057972409?thread_ts=1713806069.886019&cid=CJ0H8Q97C.) with dev-inf.